src/lib.rs

```diff
@@ -882,6 +884,18 @@ pub fn delegate(tokens: TokenStream) -> TokenStream {
             let modify_expr = |expr: &Expr| {
                 let body = if let Some(target_trait) = &attributes.target_trait {
```

Comment:
We should check somewhere that both `const` and `call` weren't used together, that doesn't make sense.

Reply:
fair ! I'll do that 👍